TechAndCheck/hypatia
# Delete files from tmp

This makes sure that temp files are deleted after uploading to amazon.

To test:
- Empty the `./tmp` folder in the project directory
- Run a scrape (using a test?)
- Ensure the folder is empty still.
